gravitee-apim-console-webui/src/scss/gio-global-style.scss

```diff
@@ -8,26 +8,42 @@
 
 $background: map.get(gio.$mat-theme, background);
 
-.gio-snack-bar-success {
-  white-space: pre-wrap;
-  background-color: mat.get-color-from-palette(gio.$mat-success-palette, default);
-  color: mat.get-color-from-palette(gio.$mat-success-palette, default-contrast);
-
-  .mat-button {
+.gio-snack-bar-success.mat-mdc-snack-bar-container {
+  .mdc-snackbar__surface {
+    white-space: pre-wrap;
+    background-color: mat.get-color-from-palette(gio.$mat-success-palette, default);
```

Comment:
The background colors can be controlled with `--mat-snack-bar-button-color`

Reply:
Done